diagnosisApplication/src/main/use_case/diagnosis/DiagnosisInteractor.java

```diff
@@ -167,37 +169,10 @@ else if (healthDiagnosisList.size() == 2) {
 
             diagnosis1Hash.put(3, issue2Hash);
 
-            DiagnosisOutputData diagnosisOutputData = new DiagnosisOutputData(diagnosis1Hash, diagnosis2Hash, diagnosis3Hash);
+            DiagnosisOutputData diagnosisOutputData = new DiagnosisOutputData(issue1Hash, issue2Hash, issue3Hash);
```

Comment:
In the future we can also then remove the diagnosis1Hash, and for the other ones as well